src/vm/appdomain.cpp

```diff
@@ -2732,7 +2732,10 @@ void SystemDomain::LoadBaseSystemClasses()
     // We have delayed allocation of mscorlib's static handles until we load the object class
     MscorlibBinder::GetModule()->AllocateRegularStaticHandles(DefaultDomain());
 
+    // used by MethodTable::ContainsStackPtr
```

Comment:
Undo (not needed once ContainsStackPtr is deleted)

Reply:
Done.